src/Core/Math/Ellipsoid.js

```diff
@@ -55,11 +56,10 @@ Ellipsoid.prototype.setSize = function setSize(size) {
     this._radiiSquared = new THREE.Vector3(size.x * size.x, size.y * size.y, size.z * size.z);
 };
 
-
+const k = new THREE.Vector3();
```

Comment:
Allocating `k` outside of `cartographicToCartesian` isn't a good idea, since you're returning it at each call.

I'd prefer that you add a `target` param and allocate a return vector if missing.

Reply:
updated